### PR TITLE
add validation pop-up for parsing

### PR DESCRIPTION
**Pull request by LennyDZ**
Add a pop-up that propose the parsing and ask for validation.

Also add a parsing pattern "tomorrow"

### CODE REVIEW

File: app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt [hunk 1]
```diff
@@ -80,13 +81,14 @@ class AddDeadlineActivity : AppCompatActivity() {
...
         updateDisplayedDateAndTime()
-        KeyboardVisibilityEvent.setEventListener(this, object : KeyboardVisibilityEventListener {
-            override fun onVisibilityChanged(isOpen: Boolean) {
-                if (!isOpen) {
-                    updateDisplayedInfoAfterTitleChange()
-                }
+        textTitle.setOnKeyListener { _, keyCode, keyEvent ->
+            if (keyEvent.action == KeyEvent.ACTION_DOWN && keyCode == KeyEvent.KEYCODE_ENTER) {
+                updateDisplayedInfoAfterTitleChange()
+                return@setOnKeyListener true
```
[the remark this answers is not in the document]
Comment (LennyDZ) [reply app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt:87]: It's for the onKeyListener, it means that the event has been consumed. [see here](https://developer.android.com/reference/android/view/View.OnKeyListener)

[the remark this answers is not in the document]
Comment (LennyDZ) [reply app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt:87]: SetOnKeyListener add a listener to keyboard event

**Review by ValentinAebi — APPROVED** [review]

File: app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt [hunk 2]
```diff
@@ -90,13 +95,22 @@ class AddDeadlineTest {
         )
     }
 
+    @Test
+    fun `parsing validation pop-up`(){
```
Comment (Matathiasq13824) [inline app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt:99]: You could just add the line to click on the ok button to increase a little the coverage and to be sure that the dialog disappear and add another test to test the cancel option but this is minimal

**Review by Matathiasq13824 — CHANGES_REQUESTED** [review]
Great code, nothing to change except my comment and maybe add some comments on the private function in add deadline

File: app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt (hunk 2 above)
Comment (LennyDZ) [reply app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt:102]: I did that

**Review by Matathiasq13824 — APPROVED** [review]

**Pull request merged**